script.js

```diff
@@ -0,0 +1,53 @@
+console.log("Long, long ago in a galaxy far, far away... ")
+function CharacterSelect() {
+  var characterSelect = prompt ("Welcome to Star Wars: The Adventure Game! Would you like to play as (L)uke Skywalker, (H)an Solo or Lando (C)arlissian? L/H/C? (case sensitive)")
+    if (characterSelect  == 'L') {
+      console.log("Luke Skywalker  - You're on the forest moon of Endor with Han Solo and Leia. Your mission is to disrupt the Death Stars defense power grid. However, your arrival has caught the attention of 2 storm troopers who race off on their Speeder Bikes to alert the base.")
+      var giveChase = prompt ("Do you give chase and stop them? Y/N (case sensitive)");
+        if (giveChase == 'Y') {
+          console.log("You chase down the stormtroopers and stop them from alerting the base of your presence. Good decision! But wait... It's a TRAP! Darth Vador has used the stormtroopers to lure you away and capture you. He now takes you to the Death Star and presents you to the emperor.")
+          var challenge = prompt ("Do you (A) Challenge the emperor to a showdown with the fate of the empire as the prize? or (B) Attempt to reach Darth Vador's sensitive side and convince him to side with his only son and turn against his old master? A/B (case sensitive)");
+            if (challenge == 'A') {
+              console.log("Bad call there Sparkie! The emperor mops the floor with you. Although Han Solo and Lando are successful in their respected missions, the emperor springs his surprise attack on the rebellion and decimates them once and for all. Nice going there!")
+            }
+            else if (challenge == 'B') {
+              console.log("All you need is love to reach your estranged dad, Darth Vador and you do so successfully. Upon realizing what a bad father figure he'd become, he decides to switch sides and help you defeat his old master, the emperor. Upon discarding the old geezers carcass down an unnecessarily placed bottomless pit, your dad takes you out for ice-cream and you play catch down on Endor as the rest of you cronies celebrate the demise of the emperor and his evil rule. The Star Wars universe salutes you!");
+            }
+          }
+        }
+        else if (giveChase == 'N') {
+          console.log("You chose to ignore the stormtroopers and they alert the base of your presence. The mission is a failure and the rebellion is decimated by the Death Star. Nice going there!")
+        }
```

Comment:
Would include an `else` here to re-prompt the user in the event they do not choose either option